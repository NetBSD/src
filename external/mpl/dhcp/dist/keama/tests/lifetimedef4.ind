# DHCPv4 use ISC DHCP default lifetimes
