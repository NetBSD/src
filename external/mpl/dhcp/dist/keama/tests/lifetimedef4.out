{
  # DHCPv4 use ISC DHCP default lifetimes
  "Dhcp4": {
    /// Use ISC DHCP default lifetime
    "valid-lifetime": 43200,
    /// Use ISC DHCP min lifetime
    "min-valid-lifetime": 300,
    /// Use ISC DHCP max lifetime
    "max-valid-lifetime": 86400
  }
}
